src/form/Form.css

```diff
@@ -0,0 +1,45 @@
+.form_Form {
+	color:#333333;
+	border-width: 1px;
+	border-color: #999999;
+	border-collapse: collapse;
+}
+
+.form_Form th {
+    padding:5px 10px;
+	background-color:#1f77b4;
+	border-width: 1px;
+	border-style: solid;
+	border-color: #a9c6c9;
+    color:white;
+    white-space: nowrap;
+}
+
+.form_Form tbody tr {
+	background-color:#dcf1ff;
+    color:black;
+}
+
+.form_Form tbody tr:hover {
+	background-color:#ff7f0e;
+    color:white;
+}
+
+.form_Form tbody td {
+	border-style: solid;
+	border-color: #a9c6c9;
+    white-space: nowrap;
+}
+
+.form_Form td {
+	border-width: 1px;
+    padding:5px;
+}
+
+.form_Form input {
+    padding:0px 0px;
+}
+
+.form_Form tfoot button{
+    margin-left:5px;
+}
```

Comment:
Indentation is inconsistent in this file (mixed / tabs and spaces) - preference is for 4 spaces instead of tabs.